puppeteer/atiAnalytics/index.test.js

```diff
@@ -513,7 +522,7 @@ const supportsAmp = ({ contentType }) =>
 const ampTestSuites = canonicalTestSuites.filter(supportsAmp).map(testSuite => {
   return {
     ...testSuite,
-    path: `${testSuite.path}.amp`,
+    path: getPath({ path: testSuite.path, suffix: '.amp' }),
```

Comment:
where is this getPath meant to come from ? are we missing an import here cant see it at the top of the file, and github makes it realllly hard to look for stuff lol - can see the tests are failing as it's allegedly undefined 😕